README.md

```diff
@@ -8,10 +8,11 @@ This code base is meant to categorize each line of a *python* script several dif
 * conditionals
 * function defs
 * assignments
-
-# To be supported
 * function calls
 * hash lines and storage
+
+# To be supported
```

Comment:
This should be labeled todo rather than to be supported... This should also maybe be a different document or should use the issue board.

Reply:
This has been relabelled todo list for now. I have not pulled out the todo as of yet.